### ISSUE TITLE
Skip archived repos in org

### ISSUE THREAD

**Issue by nitrocode** [Clever/microplane#77]
This is one of my favorite tools. Thank you very much for opening the issues tab.

It would be nice when using

```
mp init "<org>" --all-repos
```

to have a flag for `--skip-archived` so we can skip initializing archived repos.

**Comment by nathanleiby** [comment]
Thanks for the suggestion and glad you're enjoying using `microplane`! We'll keep this in mind as a future improvement, and are also open to PRs from the community.

**Comment by dbirks** [comment]
Just hit this one myself, and was thinking about trying to fix it. Archived repos are always read-only right? Maybe we should have it skip archived repos, because you wouldn't be able to push a branch to create a PR.

**Comment by nathanleiby** [comment]
Yes, archived repos in Github are read-only ([docs](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/archiving-a-github-repository)), so it makes sense to skip them completely.
